### PR TITLE
arch/nrf53: add UID support

### PR DESCRIPTION
**Pull request by raiden00pl**
## Summary
Ported from NRF52

arch/nrf53: add UID support
## Testing
CI

### CODE REVIEW

File: arch/arm/src/nrf53/nrf53_uid.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * Included Files
+ ****************************************************************************/
+
+#include <stdint.h>
+
+/****************************************************************************
+ * Public Function Prototypes
+ ****************************************************************************/
+
+void nrf53_get_uniqueid(uint8_t uniqueid[]);
```
Comment (pkarashchenko) [inline arch/arm/src/nrf53/nrf53_uid.h:34]:
```suggestion
void nrf53_get_uniqueid(uint8_t uniqueid[8]);
```
Optional, just for readability.

File: arch/arm/src/nrf53/nrf53_uid.c [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#include "arm_internal.h"
+#include "nrf53_uid.h"
+
+#include "hardware/nrf53_ficr.h"
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+void nrf53_get_uniqueid(uint8_t uniqueid[])
```
Comment (pkarashchenko) [inline arch/arm/src/nrf53/nrf53_uid.c:36]: Optional

```suggestion
void nrf53_get_uniqueid(uint8_t uniqueid[8])
```

**Review by pkarashchenko — APPROVED** [review]

File: arch/arm/src/nrf53/nrf53_uid.h (hunk 1 above)
Comment (raiden00pl) [reply arch/arm/src/nrf53/nrf53_uid.h]: done, fixed also for nrf52

**Review by pkarashchenko — APPROVED** [review]
Thank you!

**Pull request merged**